api/desecapi/models.py

```diff
@@ -522,6 +512,27 @@ class Meta:
             models.UniqueConstraint(fields=['token'], condition=Q(domain__isnull=True), name='unique_entry_null_domain')
         ]
 
+    def clean(self):
+        default_policy = self.token.get_policy(domain=None)
+        if self.pk:  # update
+            # Can't change policy's default status ("domain NULLness") to maintain policy precedence
+            if (self.domain is None) != (self.pk == default_policy.pk):
+                raise ValidationError({'domain': 'Policy precedence: A default policy must exist before others.'})
```

Comment:
error message needs to be adjusted

Reply:
I think it is accurate; please suggest a specific message

Reply:
replaced by more specific messages